native_client/deepspeech.cc

```diff
@@ -478,3 +478,41 @@ char*
 DS_Version() {
   return strdup(ds_version());
 }
+
+// Returned pointers are read only and  MUST not be deallocated or modified
+char*
+DS_ErrorCodeToErrorMessage(int aErrorCode){
+  switch(aErrorCode)
+  {
+    case DS_ERR_OK:
+      break;
```

Comment:
You have to return a valid string here, otherwise this will return a garbage value.
```suggestion
      return "No error.";
```